docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
```

Comment:
I think we should help people out a bit more here if we can.

CatalogSources _represent_ a repository of _bundles_, which are collections of resources that (today) must contain CSVs, and there are multiple implementations of a CatalogSource backend, the current recommended one is a registry image (etc etc). If we're good about defining our terms, we don't have to write everything out like this and can just link to the definitions.